### PR TITLE
[Anomaly] Move sample to root/tools

### PR DESCRIPTION
**Pull request by ashwinvaidya17**
## Description
Move sample.py to address CI issue.

## Build
Here is the [link](https://ci-ote.iotg.sclab.intel.com/job/ote/job/pr-ote-test/144/) to the CI build. 🏃‍♂️ ⏳

### CODE REVIEW

File: external/anomaly/tools/__init__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+"""
+Collection of tools to run anomaly training extension.
+"""
+
+# Copyright (C) 2021 Intel Corporation
```
Comment (samet-akcay) [inline external/anomaly/tools/__init__.py]: 2022?

**Review by samet-akcay — DISMISSED** [review]

**Review by samet-akcay — DISMISSED** [review]

**Comment by samet-akcay** [comment]
run ote_sdk tests

**Comment by ashwinvaidya17** [comment]
run ote-test

**Comment by goodsong81** [comment]
I've run a manual CI test. It passed the pre-commit code scan test as expected. But there were bunch of similar import errors regarding anomaly. I'm not sure it's related to this change, though...
https://ci-ote.iotg.sclab.intel.com/job/ote/job/pr-ote-test/122/artifact/.workspace/scm/ote-validation/test_log/tests.log
image

**Comment by goodsong81** [comment]
Passed pylint.

**Pull request merged**